### ISSUE TITLE
GitHub actions tests failing: cause and possible solution

### ISSUE THREAD

**Issue by gully** [HajimeKawahara/exojax#367]
I recently did a fresh install of ExoJAX v1.3 and encountered the same error that appears in the GitHub Actions:

`error: pandas 2.0.0rc1 is installed but pandas<2.0.0,>=1.0.0 is required by {'bqplot'}`

As a workaround, I resolved this locally by requiring `pandas=1.5.3` in my conda environment.  Alternatively pinning this requirement could go in the python setup.py file.

**Comment by HajimeKawahara** [comment]
Thanks! I added  `pandas<2.0.0,>=1.0.0` in setup.py in PR #359, which seems to have solved the error for pandas. There is still a problem with `vaex.hdf` though 
```
>       from vaex.hdf5.writer import Writer
E       ModuleNotFoundError: No module named 'vaex.hdf5'
```
 I've never encountered this `vaex.hdf` error in my local environments.

**Comment by HajimeKawahara** [comment]
(Temporary) solved in #392 

See also https://github.com/vaexio/vaex/issues/2376
